### PR TITLE
Add missing v prefix to ppx_let lower bound

### PR DESCRIPTION
**Pull request by kevinji**
I think this might fix the build issue, but will wait for Travis to finish running.

### CODE REVIEW

**Comment by kevinji** [comment]
@seliopou Hmm so using ppx_let >= v0.14.0 eventually requires OCaml >= 4.07.0 which breaks the 4.06 build. Do you want to drop 4.06 from the Travis build and maybe add 4.11 to round things out?

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: c
 sudo: false
+dist: focal
```
Comment (seliopou) [inline .travis.yml:3]: What does this do?

Comment (kevinji) [reply .travis.yml:3]: This changes the Ubuntu distribution used for Travis, which was one of the suggestions in ocaml/ocaml-ci-scripts#351 for handling the build error (see the failed build log on the first commit).

**Comment by seliopou** [comment]
Thanks! I'll fix the OCaml version constraint separately.

**Pull request merged**